tests/make_failure_report.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Generates a summary plot comparison for each failed test case
+
+OPM_TESTS_ROOT=$1
+BUILD_DIR=$2
+RESULT_DIR=$3
+SOURCE_DIR=`dirname "$0"`
+
+FAILED_TESTS=`cat $BUILD_DIR/Testing/Temporary/LastTestsFailed*.log`
+
+mkdir -p $2/failure_report
+cd $2/failure_report
```

Comment:
Tiny nit: You already store `$2` in `BUILD_DIR`.  Might it perhaps be more descriptive to replace `$2` with `$BUILD_DIR` here?

Reply:
done.